thirdparty/pybind11/include/pybind11/pytypes.h

```diff
@@ -292,7 +292,7 @@ class error_already_set : public std::runtime_error {
     /// Constructs a new exception from the current Python error indicator, if any.  The current
     /// Python error indicator will be cleared.
     error_already_set() : std::runtime_error(detail::error_string()) {
-        PyErr_Fetch(&type.ptr(), &value.ptr(), &trace.ptr());
+        //PyErr_Fetch(&type.ptr(), &value.ptr(), &trace.ptr());
```

Comment:
Don't we need this?

Reply:
I created a follow on ticket to explore this further -- but this was causing problems. I read several old tickets where people removed this line in regards to triggering seg faults. Usually this is an issue elsewhere, but ExecuteScript was causing problems. In most cases it was fine because we weren't really handling python exceptions, but I created a follow on because at some point I would like to better deal with that.